docs/tutorials/ifci_log_qgroundcontrol.rst

```diff
@@ -0,0 +1,95 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_log:
+
+********************************************
+Exporting IFCI Logs Using QGroundControl
+********************************************
+
+IFCI data can be found in the flight controller logs. These logs can be exported from the flight controller using QGroundControl.
+
+Click the QGroundControl icon on the upper left-hand corner of the application.
+Then click "Analyze Tools" from the pop-up menu.
+
+.. figure:: ../_static/tutorial_images/ifci_log/analyze_tools.png
+    :align: center
+    :alt: Analyze Tools
+
+    QGroundControl Analyze Tools
+
+
+Initially, there may be no logs displayed. If so, click the "Refresh" button to display the logs available on your flight controller.
```

Comment:
Is this how it works with ardupilot as well? I honestly don't know, but if it isn't I want to make sure we say that this is PX4 specific.